classes/fields/file.php

```diff
@@ -547,6 +543,7 @@ public function build_dfv_field_options( $options, $args ) {
 					'method'   => 'upload',
 					'pod'      => $pod_id,
 					'field'    => $field_id,
+					'item_id'  => $item_id,
```

Comment:
For backcompat on any hooks that were looking for `post_id`

```suggestion
					'item_id'  => $item_id,
					'post_id'  => $item_id,
```